rules/opentitan_test.bzl

```diff
@@ -502,10 +502,6 @@ def opentitan_functest(
             flash = rom
         else:
             target_data.append(flash)
-            if target in ["fpga_cw310_rom", "fpga_cw310_test_rom"]:
```

Comment:
Just checking this is not an issue, since this rule is pretty brittle, and this is failing private CI right now. Why did you remove this?

Reply:
IIRC I couldn't find references to these and they seemed stale. Thanks for checking the logs and pointing me to this hunk.